templates/components/common/payment-icons.html

```diff
@@ -1,17 +1,21 @@
-<div class="footer-payment-icons">
-    {{#if theme_settings.show_accept_amex}}
-        <svg class="footer-payment-icon"><use xlink:href="#icon-logo-american-express"></use></svg>
-    {{/if}}
-    {{#if theme_settings.show_accept_discover}}
-        <svg class="footer-payment-icon"><use xlink:href="#icon-logo-discover"></use></svg>
-    {{/if}}
-    {{#if theme_settings.show_accept_mastercard}}
-        <svg class="footer-payment-icon"><use xlink:href="#icon-logo-mastercard"></use></svg>
-    {{/if}}
-    {{#if theme_settings.show_accept_paypal}}
-        <svg class="footer-payment-icon"><use xlink:href="#icon-logo-paypal"></use></svg>
-    {{/if}}
-    {{#if theme_settings.show_accept_visa}}
-        <svg class="footer-payment-icon"><use xlink:href="#icon-logo-visa"></use></svg>
-    {{/if}}
-</div>
+{{#with theme_settings}}
```

Comment:
@bc-julianleong @bc-ranji : updated so that the footer-payment-icons only shows if at least one of the payment icons should show

before:
![screen shot 2016-12-12 at 6 25 28 pm](https://cloud.githubusercontent.com/assets/1357197/21125153/57323294-c098-11e6-837d-606735299eb3.png)

after:
![screen shot 2016-12-12 at 6 25 00 pm](https://cloud.githubusercontent.com/assets/1357197/21125154/5a06f374-c098-11e6-980f-929b493e01aa.png)